### PR TITLE
Add integration specs back as part of rake ci

### LINKED ISSUE #121: rake ci does not run integration specs anymore

### PR DESCRIPTION
**Pull request by mbj**
[fix #121]

### CODE REVIEW

**Comment by mbj** [comment]
@backus Ready for review.

File: tasks/metrics/ci.rake [hunk 1]
```diff
@@ -3,6 +3,7 @@ task ci: %w[ci:metrics metrics:mutant]
 
 namespace :ci do
   tasks = %w[
+    spec:integration
```
Comment (backus) [inline tasks/metrics/ci.rake]: Maybe it would help to place this second in the task prerequisites? This way CI fails much faster if unit tests fail

Comment (mbj) [inline tasks/metrics/ci.rake]: I hate having to thing sequentially. I want all this to happen in parallel. But yeah I think you are right integration tests should come after the fast feedback tasks.

Comment (backus) [inline tasks/metrics/ci.rake]: I agree once this is parallelized

**Comment by mbj** [comment]
@backus Ready for the next pass.

**Comment by backus** [comment]
LGTM assuming build passes

**Pull request merged**